setup.cfg

```diff
@@ -0,0 +1,22 @@
+[metadata]
+name = zppy-e3sm-chem-diags
```

Comment:
This is a very important point for us to decide on now before there is a release: what should the name of the python package be?  This was my suggestion but it could also just be `e3sm-chem-diags` or `e3sm_chem_diags`.  Please let me know what you prefer.

Reply:
Currently, there is a directory `zppy_e3sm_chem_diags` where I have moved a lot of files.  This is the python package itself.  Any files outside of this directory are in the repository but not in the python package.  So keep that in mind.  None of them will be available to `zppy` or to users that have installed the conda package.

Also, I will note that we will rename `zppy_e3sm_chem_diags` to correspond to the name you choose here.  Hyphens are not allowed in the directory name (or any directories or python files in a python package, since they get confused with minus signs in python imports).  So it is common for the package name to have hyphens but the directory to have underscores like in my example.  But both could have underscores if you prefer.

The name of the python package should be all lowercase as I have here (so shouldn't be ChemDyg, for example).

Reply:
Since python only allows underscore in the directory name, I'd prefer only use underscore in the names of directory and python package for consistency.

The name we came up for the E3SM chemistry diagnostic package is `ChemDyg` as the github repo name. I think we would like to use `ChemDyg` for the python package name as well.

With the all lowercase requirement, I am leaning towards `chemdyg` as the python package name, which will need these changes:

- Rename the directory `zppy_e3sm_chem_diags` to a new name. Any suggestions for the new directory name? When cloning from github, we will have the `ChemDyg` directory, under which is the python package directory. Is there a standard naming convention? If not, what about `chemdyg_package` (I feel it's a bit weird if we have `chemdyg` under `ChemDyg`)?
- Change to `name = chemdyg` in this setup.cfg file

Welcome to suggestions.

Reply:
Sorry that I had a few meetings this morning and couldn't add my two cents here earlier. Please see my suggestions on the python package name and directory name above.

Reply:
If so, I prefer to rename the directory to "chemdyg_package". 
How about the file name: e3sm_chem_diags.py? Should it change to "chemdyg.py" or "chemdyg_package.py" as well?

Reply:
I like `chemdyg.py`. Certainly not `chemdyg_package.py`.

Reply:
Nearly all python package name the directory the same as the package name.  So it would be appropriate to name the package `chemdyg` and the directory `chemdyg` but it would be very confusing to a python programmer if you called the directory `chemdyg_package`, since that's quite non-standard.  Some people call the directory something like `src` or `lib` but I think that's pretty poor form and doesn't make things less confusing.  I would strongly recommend just `chemdyg` for both.

I will day that `dyg` doesn't mean anything to me so it would be helpful to have some explanation of what that is intended to mean.  Somehow short for "diagnostics"?

Reply:
I do think `e3sm_chem_diags.py` be named either `chemdyg.py`.  That somewhat depends on what zppy will require when it is looking for plugins.  Another possibility would be to put it in `__init__.py`.  That would be intuitive for a python package programmer but won't make much sense to people new to python packaging.  So I won't recommend that approach.

Reply:
@xylar , we should follow the common practice to use the same name `chemdyg` for the package and the directory. I suggested the "_package" suffix because git clone will create a `chemdyg` directory if the users do not give it a different name. I am fine with the path like `chemdyg/chemdyg`.

Yes, `dyg` stands for diagnostics. We can describe it in the README.md file.

It seems that we all think it is a good idea to rename the python file to `chemdyg.py`.